src/vcpkg/build.cpp

```diff
@@ -182,16 +182,13 @@ namespace vcpkg::Build
         // Build only takes a single package and all dependencies must already be installed
         const ParsedArguments options = args.parse_arguments(COMMAND_STRUCTURE);
         BinaryCache binary_cache{args, paths};
+
         bool default_triplet_used = false;
         const FullPackageSpec spec = check_and_get_full_package_spec(options.command_arguments[0],
                                                                      default_triplet,
                                                                      default_triplet_used,
```

Comment:
Seems like after the merge there is more cleanup to be done. Pretty sure `default_triplet_used` can be removed completely.